### PR TITLE
DOCSP-17144 compatibility via snippet

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-17144-compatibility-via-snippet/reference/compatibility/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-17144)

### CODE REVIEW

File: source/reference/compatibility.txt [hunk 1]
```diff
@@ -10,8 +10,14 @@ Compatibility Changes with Legacy ``mongo`` Shell
    :depth: 1
    :class: singlecol
 
-The following behaviors in ``mongosh`` affect compatibility with
-operations performed in the legacy ``mongo`` shell.
+This page addresses differences between ``mongosh`` and the legacy
```
Comment (jason-price-mongodb) [inline source/reference/compatibility.txt:13]: "This page describes ..."

Comment (davemungo) [reply source/reference/compatibility.txt]: Done

File: source/reference/compatibility.txt [hunk 2]
```diff
@@ -10,8 +10,14 @@ Compatibility Changes with Legacy ``mongo`` Shell
...
+``mongo`` shell. In addition to the alternatives listed here, you can
+use the `mongocompat
+<https://github.com/mongodb-labs/mongosh-snippets/blob/main/snippets/mongocompat/index.js>`__
+snippet to access to legacy :binary:`mongo` shell APIs. For more
```
[the remark this answers is not in the document]
Comment (davemungo) [reply source/reference/compatibility.txt:17]: It's ok in this context - "snippets" are a mini feature in mongosh, sort of like plug-ins or extensions.

**Comment by jason-price-mongodb** [comment]
@davemungo LGTM.

**Comment by davemungo** [comment]
@mmarcon would you take a look at this PR when you have a moment?

File: source/reference/compatibility.txt [hunk 3]
```diff
@@ -10,8 +10,14 @@ Compatibility Changes with Legacy ``mongo`` Shell
    :depth: 1
    :class: singlecol
 
-The following behaviors in ``mongosh`` affect compatibility with
-operations performed in the legacy ``mongo`` shell.
+This page describes differences between ``mongosh`` and the legacy
+``mongo`` shell. In addition to the alternatives listed here, you can
+use the `mongocompat
+<https://github.com/mongodb-labs/mongosh-snippets/blob/main/snippets/mongocompat/index.js>`__
```
Comment (mmarcon) [inline source/reference/compatibility.txt:16]:
```suggestion
<https://github.com/mongodb-labs/mongosh-snippets/tree/main/snippets/mongocompat>`__
```
I'd link to the root folder in the repo where the README file is.

File: source/reference/compatibility.txt [hunk 4]
```diff
@@ -10,8 +10,14 @@ Compatibility Changes with Legacy ``mongo`` Shell
...
+snippet to access to legacy :binary:`mongo` shell APIs. For more
+information, see :ref:`snip-overview`.
+
+snippet install analyze-schema
```
Comment (mmarcon) [inline source/reference/compatibility.txt:20]:
```suggestion
snippet install mongocompat
```
Can we show this as a code snippet?

**Review by mmarcon — CHANGES_REQUESTED** [review]

**Review by davemungo — COMMENTED** [review]
@mmarcon thanks for your feedback.

This is ready for another pass.

**Review by addaleax — APPROVED** [review]

**Pull request merged**